x-pack/plugins/observability_solution/asset_manager/server/lib/entities/create_and_install_ingest_pipeline.ts

```diff
@@ -6,15 +6,15 @@
  */
 
 import { ElasticsearchClient, Logger } from '@kbn/core/server';
-import { OAMDefinition } from '@kbn/oam-schema';
+import { EntityDefinition } from '@kbn/oam-schema';
```

Comment:
lingering ref to OAM here (and the same in a few other places)

Reply:
Whoops... When I started working on the "optional identity fields" change I realized I forgot to push renaming the `kbn-oam-schema` to `kbn-entities-schema` changes I had made.